src/hooks/useCalculatedColumns.ts

```diff
@@ -15,26 +15,31 @@ interface ColumnMetric {
   left: number;
 }
 
+const DEFAULT_COLUMN_WIDTH = 'auto';
+const DEFAULT_COLUMN_MIN_WIDTH = 80;
+
 interface CalculatedColumnsArgs<R, SR> extends Pick<DataGridProps<R, SR>, 'defaultColumnOptions'> {
   rawColumns: readonly Column<R, SR>[];
   rawGroupBy: Maybe<readonly string[]>;
   viewportWidth: number;
   scrollLeft: number;
   columnWidths: ReadonlyMap<string, number>;
+  flexColumnWidths: ReadonlyMap<string, number>;
   enableVirtualization: boolean;
 }
 
 export function useCalculatedColumns<R, SR>({
   rawColumns,
   columnWidths,
+  flexColumnWidths,
   viewportWidth,
   scrollLeft,
   defaultColumnOptions,
   rawGroupBy,
   enableVirtualization
 }: CalculatedColumnsArgs<R, SR>) {
   const defaultWidth = defaultColumnOptions?.width;
```

Comment:
Should we do this here instead?
```suggestion
  const defaultWidth = defaultColumnOptions?.width ?? DEFAULT_COLUMN_WIDTH;
```